README.md

```diff
@@ -1,5 +1,15 @@
+# DEPRECATED
+> [!WARNING]  
+> This is no longer supported as native Gnosis Chain support has been introduced. Please consider using the alternative setup path provided is [official documentation](https://docs.gnosischain.com/node/manual/beacon/nimbus).
+
+
+```shell
 docker build -t gnosischain/nimbus:latest --build-arg UPSTREAM_VERSION=v2.2.2 .
-docker push gnosischain/nimbus:latest   
+```
+
+```shell
+docker push gnosischain/nimbus:latest
+```
```

Comment:
Why change this?

Reply:
This were the plain text docker cli commands intended to be formatted like that